inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -141,12 +141,28 @@ def _get_hep_record_brief(hep_record):
     if abstract is not None:
         brief['abstract'] = abstract
 
+    arxiv_eprint = get_value(hep_record, 'arxiv_eprints[0].value')
+    if arxiv_eprint is not None:
+        brief['arxiv_eprint'] = arxiv_eprint
+
+    number_of_pages = get_value(hep_record, 'number_of_pages')
+    if number_of_pages is not None:
+        brief['number_of_pages'] = number_of_pages
+
     authors = hep_record.get('authors')
     if authors is not None:
         author_briefs = []
         for author in authors[:3]:
             author_briefs.append({'full_name': author['full_name']})
         brief['authors'] = author_briefs
+
+    public_notes = hep_record.get('public_notes')
+    if public_notes is not None:
+        public_notes_value = []
+        for public_note in public_notes:
+            public_notes_value.append(public_note['value'])
```

Comment:
just be more future proof and closed to actual schema, it is better to have a dict list like `{'value': public_note['value']}`